### PR TITLE
Add required compiler flag to gearmand-0.34-r1.ebuild

### PR DESCRIPTION
**Pull request by Szpadel**
Germand not compiling in compilers that not use c++11 by default (eg. gcc < 5.0)
Enabling this mode explicitly in ebuild.

### CODE REVIEW

File: sys-cluster/gearmand/gearmand-0.34-r1.ebuild [hunk 1]
```diff
@@ -51,6 +51,9 @@ src_configure() {
 		# preprocessor then.
 		append-cppflags -DDEBUG
 	fi
+	
+	# Explicitly enable c++11 mode
+	append-cppflags -std=c++11
```
Comment (mgorny) [inline sys-cluster/gearmand/gearmand-0.34-r1.ebuild]: Strictly speaking, that's not a preprocessor's flag :-P.

**Comment by mgorny** [comment]
@gentoo/proxy-maint

**Comment by mgorny** [comment]
@Szpadel, did you see my comment? It should be a cxxflag, not cppflag.

File: sys-cluster/gearmand/gearmand-0.34-r1.ebuild [hunk 2]
```diff
@@ -51,6 +51,9 @@ src_configure() {
 		# preprocessor then.
 		append-cppflags -DDEBUG
 	fi
+
```
Comment (mgorny) [inline sys-cluster/gearmand/gearmand-0.34-r1.ebuild]: If you did `git log -p`, you'd notice red block saying you've introduced whitespace issue here. But I'm going to fix it for you.

**Comment by mgorny** [comment]
Fixed whitespace for you, rebased, reformatted commit message and it's in:

```commit d7735a14ce260b2d8cc8b004949169ee2a27d272
Author: Szpadel <piotrekrogowski@gmail.com>
Date:   Tue Sep 29 10:44:19 2015 +0200

    sys-cluster/gearmand: Add -std=c++11 flag, bug #503538
    
    Fixes: https://bugs.gentoo.org/show_bug.cgi?id=503538
    Pull-Request: https://github.com/gentoo/gentoo/pull/142
```

I hope you tested it since it fails on graphite optimizations for me. But I guess that's compiler's fault.

**Comment by idella** [comment]
Once this became a pure pull req and the bug 503538 was abandoned I lost track of it.  This was up to the proxy-maintainers' members to commit.

**Pull request closed without merging**